### ISSUE TITLE
Brunch not compiling assets to priv/static

### ISSUE THREAD

**Issue by maxmouchet** [phoenixframework/phoenix#1667]
Hi,

So when I run `brunch build` in a new project, it doesn't seems to compile and copy the static assets from `web/static` to `priv/static`. When visiting the page, it shows `no route found for GET /css/app.css`, and so on, errors.
### Environment

* Elixir version (elixir -v): `1.2.4`
* Phoenix version (mix deps): `1.1.4`
* NodeJS version (node -v): `v5.11.0`
* NPM version (npm -v): `3.8.6`
* Operating system: `OS X 10.11`

### Expected behavior

```bash
% brunch build
24 Apr 21:52:52 - info: compiled 5 files into 2 files, copied 3 in 1.2 sec

% ls priv/static
css         favicon.ico images      js          robots.txt
```

### Actual behavior

```bash
~/Downloads % mix phoenix.new sample_app
# ...

Fetch and install dependencies? [Yn]
* running mix deps.get
* running npm install && node node_modules/brunch/bin/brunch build

~/Downloads/sample_app % brunch build
25 Apr 01:52:05 - info: compiled in 961ms

~/Downloads/sample_app % ls priv/static
ls: priv/static: No such file or directory
```

### Brunch verbose output

```bash
~/Downloads/sample_app % brunch b -d
  brunch:config Trying to load brunch-config +0ms
  brunch:plugins Loaded plugins: babel-brunch, css-brunch, javascript-brunch +719ms
  brunch:watch add package.json +37ms
  brunch:watch add brunch-config.js +1ms
  brunch:watch add test/test_helper.exs +6ms
  brunch:list Reading test/test_helper.exs +1ms
  brunch:watch add test/support/channel_case.ex +5ms
  brunch:list Reading test/support/channel_case.ex +0ms
  brunch:watch add test/support/conn_case.ex +1ms
  brunch:list Reading test/support/conn_case.ex +0ms
  brunch:watch add test/support/model_case.ex +0ms
  brunch:list Reading test/support/model_case.ex +1ms
  brunch:watch add test/controllers/page_controller_test.exs +0ms
  brunch:list Reading test/controllers/page_controller_test.exs +0ms
  brunch:watch add test/views/error_view_test.exs +0ms
  brunch:list Reading test/views/error_view_test.exs +1ms
  brunch:watch add test/views/layout_view_test.exs +0ms
  brunch:list Reading test/views/layout_view_test.exs +0ms
  brunch:watch add test/views/page_view_test.exs +0ms
  brunch:list Reading test/views/page_view_test.exs +0ms
  brunch:write Writing 0/0 files +76ms
25 Apr 01:57:42 - info: compiled in 1.7 sec
```

**Comment by Gazler** [comment]
Could you try using the version of `brunch` that is installed locally?

    ./node_modules/.bin/brunch build

**Comment by chrismccord** [comment]
In addition to what gazer said, also please try node 5.3 and report back. Thanks!

**Comment by aforward** [comment]
It looks like a flurry of brunch releases.  I tried aginst the latest 2.7.3 and it worked as expected, but 2.7.2 was not working.

https://github.com/brunch/brunch/releases

Trying to get more proof, but in the interim try updated brunch

````
npm update brunch
```

and trying again

**Comment by aforward** [comment]
Yup; looks like brunch.  Here's a project showing that 2.7.2 isn't properly finding the assets, but 2.7.3 is.  Looks like an unfortunate set of bad timing.

With 2.7.2

<img width="394" alt="screen shot 2016-04-25 at 7 29 37 pm" src="https://cloud.githubusercontent.com/assets/48086/14802145/f659a906-0b1c-11e6-99d1-0ed2209ecbf3.png">

With 2.7.3

<img width="1143" alt="screen shot 2016-04-25 at 7 31 10 pm" src="https://cloud.githubusercontent.com/assets/48086/14802149/fb73ca98-0b1c-11e6-90dc-cf8f11d8c2eb.png">

Here's the project with three commits to reproduce if you so care

https://github.com/aforward/aaarg

**Comment by josevalim** [comment]
Yeah, we should always be specific with our brunch dependencies, with exact
matches, because it does have regressions from time to time.


-- 


*José Valim*
www.plataformatec.com.br
Skype: jv.ptec
Founder and Director of R&D

**Comment by maxmouchet** [comment]
Indeed updating brunch solved it, tried with `2.7.4` and it's working as expected.
